packages/itwin/ec3-widget/src/hooks/useValidator.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useState } from "react";
+import SimpleReactValidator from "simple-react-validator";
+
+export const Validators = {
+  NoDuplicateUnderscore: {
```

Comment:
Do these need to start with an uppercase letter?

Reply:
Unnecessary validators were removed. Only required validator is left.